boards/arm/lpcxpresso55s69/lpcxpresso55s69_cpu0.dts

```diff
@@ -109,3 +109,23 @@
 zephyr_udc0: &usbhs {
```

Comment:
Should we add this change to `lpcxpresso55s69_ns.dts`

Reply:
Good point. Should the config option be also added to Kconfig.defconfig.lpc55S69_cpu1? I am guessing that the ctimers are also addressable for cpu1?

Also since the compile support for the imxrt6xx group of chips was added it should probably be added into the code. I don't actually know which specific boards are based on the imxrt6xx which is why I haven't added the configuration details for the ctimer/counter to be enabled with them.

Reply:
Don't worry about rt6xx in the main repo yet, we will follow up with another PR. I asked you to go ahead and extend your hal_nxp PR to include rt6xx now because that change is really simple and then we won't have to update west.yml again in the follow-up PR.

Reply:
I am not sure if we should add to cpu1. For now we can add to _ns.dts file. Also could you please rebase to the latest mainline code to see if it addresses the CI build error.